superdesk/publish/subscribers.py

```diff
@@ -121,6 +130,7 @@ def get(self, req, lookup):
 
     def on_create(self, docs):
         for doc in docs:
+            self._apply_schedule_status(doc)
```

Comment:
I'm not sure if it should actually do this on create or update, might be confusing that you select certain status explicitly and it would change it right away